core/test/routing.spec.js

```diff
@@ -546,5 +547,92 @@ describe('Routing', () => {
       component.set({ viewUrl: 'http://otherurl.de/app.html!#/someUrl' });
       assert.isTrue(routing.isNotSameDomain(config, component));
     });
+
+    describe('defaultPathSegments', () => {
+      const getDefaultPathSegment = routing.__get__('getDefaultPathSegment');
+
+      it('should return first child if no dps set', () => {
```

Comment:
I would write always default path segment instead of dps, since this is not straightforward for everybody